config/cmake/H5pubconf.h.in

```diff
@@ -604,11 +604,11 @@
 /* Define using v1.14 public API symbols by default */
 #cmakedefine H5_USE_114_API_DEFAULT @H5_USE_114_API_DEFAULT@
 
-/* Define using v1.16 public API symbols by default */
-#cmakedefine H5_USE_116_API_DEFAULT @H5_USE_116_API_DEFAULT@
+/* Define using v2.0.0 public API symbols by default */
+#cmakedefine H5_USE_200_API_DEFAULT @H5_USE_200_API_DEFAULT@
 
-/* Define using v1.18 public API symbols by default */
-#cmakedefine H5_USE_118_API_DEFAULT @H5_USE_118_API_DEFAULT@
+/* Define using v2.1.0 public API symbols by default */
+#cmakedefine H5_USE_210_API_DEFAULT @H5_USE_210_API_DEFAULT@
```

Comment:
From external discussions: if we assume that we're going to continue tying file format changes to major releases, it makes sense that the next API version should be 3.0 since the API wouldn't change in an incompatible way for a minor release (v2.0.0 -> v2.1.0). This also has the benefit of keeping the library version bounds a bit more sane (at least for a while), as they'll go V200 -> V300 -> V400 rather than v200 -> v210 ... -> v2110 (for a 2.1.10 release).

Reply:
I think the names should be `major_minor`, so `H5_USE_2_0_API_DEFAULT`